components/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt/src/main/java/org/wso2/carbon/identity/api/resource/mgt/constant/SQLConstants.java

```diff
@@ -65,10 +65,18 @@ public class SQLConstants {
             "DESCRIPTION, TENANT_ID, TYPE, REQUIRES_AUTHORIZATION FROM API_RESOURCE WHERE ";
     public static final String GET_API_RESOURCES_TAIL =
             " (TENANT_ID = %d OR TENANT_ID IS NULL) ORDER BY CURSOR_KEY %s LIMIT %d";
+    public static final String GET_API_RESOURCES_TAIL_FOR_ORGANIZATIONS =
+            " (TENANT_ID = %d OR TENANT_ID IS NULL) AND TYPE != 'TENANT' AND TYPE != 'SYSTEM' " +
```

Comment:
Shall we use the equal conditions for types rather than not equal conditions because if we add another type for root organization those will be reflected in organization API responses.

Reply:
Also can we go for this pattern for more readability TYPE NOT IN ('TENANT', 'SYSTEM') or
TYPE IN ('<X>', '<Y>')

Reply:
Does sub orgs need "CONSOLE_FEATURE" ? I thought its only `CONSOLE_ORG_FEATURE`
Instead of euqlas can't we go with TYPE IN () pattern?

Reply:
Updated the queries with the suggestion